static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTree.tsx

```diff
@@ -67,6 +67,7 @@ export function useVirtualizedTree<T extends TreeLike>(
     scrollTop: 0,
     overscroll: props.overscroll ?? DEFAULT_OVERSCROLL_ITEMS,
     scrollHeight: props.scrollContainer?.getBoundingClientRect()?.height ?? 0,
+    maxScrollableHeight: (props.scrollContainer?.scrollHeight ?? 24) - 24, // Need subtract padding to get true height
```

Comment:
As far as I know, there doesn't seem to be a way to directly get the scrollHeight excluding padding (which is what we need here). I had to subtract the padding height here, not the cleanest solution.

Reply:
removed vertical padding from view hierarchy so this is no longer needed